### ISSUE TITLE
Per-band parameters?

### ISSUE THREAD

**Issue by JulienPeloton** [maxmahlke/phunk#1]
Hi @maxmahlke ! 

No matter if I specify the bands, the fitted parameters are always unique:

```python
from phunk import PhaseCurve
import requests
import pandas as pd
import io

r = requests.post(
  "https://fink-portal.org/api/v1/sso",
  json={
    "n_or_d": "5209",
    "withEphem": True,
    "output-format": "json"
  }
)

# Format output in a DataFrame
pdf = pd.read_json(io.BytesIO(r.content))

pc = PhaseCurve(mag=pdf["i:magpsf_red"], phase=pdf["Phase"], band=pdf["i:fid"])

pc.fit(["HG1G2"])

pc.HG1G2.PARAMS
# ['H', 'G1', 'G2']
```

What did I miss?

**Comment by maxmahlke** [comment]
For now, only the `sHG1G2` fit function separates the observations by band. I had not considered to implement this for the other models as well as they don't benefit from having multiple bands and it would be better (more obvious what's happening) if the user separates the observations by band and creates multiple, single-band phase curves.

But maybe being able to drop all observations into one `PhaseCurve` and have `phunk` take care of it is also fine, if we find a syntax that makes sure the resulting parameters are discoverable. TBD

**Comment by maxmahlke** [comment]
Fixed in version 0.1.2.
